analyzer/src/main/kotlin/managers/GradleDependencyHandler.kt

```diff
@@ -96,14 +95,7 @@ class GradleDependencyHandler(
                         e.collectMessagesAsString()
             )
 
-            Package.EMPTY.copy(
-                id = Identifier(
-                    type = "Maven",
-                    namespace = dependency.groupId,
-                    name = dependency.artifactId,
-                    version = dependency.version
-                )
-            )
+            null
```

Comment:
Commit message:

> listed in the dependency graph

How about "referenced in the dependency graph" to hint towards a `PackageReference`?

> the project's list of packages.

It's no "*the* project's list of packages" as the packages list is shared across all projects. I'd just say "list of packages".